static/app/views/eventsV2/savedQuery/discoverQueryMenu.tsx

```diff
@@ -0,0 +1,57 @@
+import {MouseEvent} from 'react';
+import styled from '@emotion/styled';
+import classNames from 'classnames';
+
+import Button from 'app/components/button';
+import DropdownMenu from 'app/components/dropdownMenu';
+import {IconEllipsis} from 'app/icons';
+import space from 'app/styles/space';
+
+const DiscoverQueryMenu = ({children}) => (
```

Comment:
This is more of a design comment, since this is a header button, the dropdown may have to be styled a bit differently. For example, other dropdowns from buttons on the header look kinda like this:
![Screen Shot 2021-08-24 at 4 20 33 PM](https://user-images.githubusercontent.com/63818634/130684457-99908183-d632-420f-b972-0352c1363170.png)

while this dropdown looks like this:
![Screen Shot 2021-08-24 at 4 22 26 PM](https://user-images.githubusercontent.com/63818634/130684539-a493949a-33d1-4fdc-b147-a7fe1a8177b7.png)

which is more like what the context menus look like in the query cards/dashboard cards